packages/apollos-ui-media-player/src/Container.tsx

```diff
@@ -23,11 +23,22 @@ import useSourceComponents from './useSourceComponents';
 
 interface ContainerProps extends IPlayerMedia, FullScreenSlidingPlayerProps {
   autoplay?: Boolean;
+  onPlay?: Function;
+  onPause?: Function;
+  onEnd?: Function;
 
   /** The Player Component. Defaults to FullscreenSlidingPlayer */
   PlayerComponent?: React.FunctionComponent;
 }
 
+const usePrevious = (value: any) => {
+  const ref = React.useRef(value);
+  React.useEffect(() => {
+    ref.current = value;
+  });
```

Comment:
```suggestion
  }, [value]);
```
Should `value` be an argument here?